<ui version="4.0" >
 <class>ParamSetClass</class>
 <widget class="QWidget" name="ParamSetClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>652</width>
    <height>446</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>ParamSet</string>
  </property>
  <widget class="QPushButton" name="pushButton" >
   <property name="geometry" >
    <rect>
     <x>570</x>
     <y>400</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text" >
    <string>Close(&amp;C)</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>10</y>
     <width>311</width>
     <height>121</height>
    </rect>
   </property>
   <property name="title" >
    <string>Common Parameter</string>
   </property>
   <widget class="QWidget" name="layoutWidget" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>80</y>
      <width>291</width>
      <height>25</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QLabel" name="label_13" >
       <property name="text" >
        <string>OutPutDir</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="txtOutDir" >
       <property name="text" >
        <string>C:/Tmp</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnSelect" >
       <property name="text" >
        <string>Select</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>50</y>
      <width>291</width>
      <height>22</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QLabel" name="label_8" >
       <property name="text" >
        <string>Pad Value Range</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="txtPadValue_1" >
       <property name="text" >
        <string>350</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_9" >
       <property name="text" >
        <string>-</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="txtPadValue_2" >
       <property name="text" >
        <string>2000</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>20</y>
      <width>291</width>
      <height>22</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QLabel" name="label_14" >
       <property name="text" >
        <string>threshold Range Value</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="txtThreshVal_1" >
       <property name="text" >
        <string>80</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_15" >
       <property name="text" >
        <string>-</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="txtThreshVal_2" >
       <property name="text" >
        <string>255</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="chkEffect" >
       <property name="text" >
        <string>Effect</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>270</y>
     <width>311</width>
     <height>121</height>
    </rect>
   </property>
   <property name="title" >
    <string>Brightness Parameter</string>
   </property>
   <widget class="QWidget" name="layoutWidget" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>20</y>
      <width>159</width>
      <height>22</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QLabel" name="label_7" >
       <property name="text" >
        <string>Scale-Z</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="txtScaleZ_2" >
       <property name="text" >
        <string>30</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QCheckBox" name="chkBar_2" >
    <property name="geometry" >
     <rect>
      <x>180</x>
      <y>20</y>
      <width>111</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text" >
     <string>BarChar</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="chkMode" >
    <property name="geometry" >
     <rect>
      <x>11</x>
      <y>52</y>
      <width>173</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text" >
     <string>OffSetMode</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3" >
   <property name="geometry" >
    <rect>
     <x>330</x>
     <y>10</y>
     <width>311</width>
     <height>121</height>
    </rect>
   </property>
   <property name="title" >
    <string>3D Graph Common Parameter</string>
   </property>
   <widget class="QWidget" name="layoutWidget" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>20</y>
      <width>131</width>
      <height>22</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QLabel" name="label_4" >
       <property name="text" >
        <string>Rotation-X</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="txtRotation_x" >
       <property name="text" >
        <string>30</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_2" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>50</y>
      <width>131</width>
      <height>22</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QLabel" name="label_5" >
       <property name="text" >
        <string>Rotation-Y</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="txtRotation_y" >
       <property name="text" >
        <string>0</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_3" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>80</y>
      <width>131</width>
      <height>22</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QLabel" name="label_6" >
       <property name="text" >
        <string>Rotation-Z</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="txtRotation_z" >
       <property name="text" >
        <string>15</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_8" >
    <property name="geometry" >
     <rect>
      <x>160</x>
      <y>50</y>
      <width>121</width>
      <height>22</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QLabel" name="label_16" >
       <property name="text" >
        <string>AvgY</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="avgy" >
       <property name="value" >
        <number>2</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget" >
    <property name="geometry" >
     <rect>
      <x>160</x>
      <y>20</y>
      <width>121</width>
      <height>22</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QLabel" name="label" >
       <property name="text" >
        <string>AvgX</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="avgx" >
       <property name="value" >
        <number>3</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_4" >
   <property name="geometry" >
    <rect>
     <x>330</x>
     <y>140</y>
     <width>311</width>
     <height>121</height>
    </rect>
   </property>
   <property name="title" >
    <string>Resolution Parameter</string>
   </property>
   <widget class="QWidget" name="layoutWidget_4" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>20</y>
      <width>159</width>
      <height>22</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QLabel" name="label_10" >
       <property name="text" >
        <string>Scale-Z</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="txtScaleZ_3" >
       <property name="text" >
        <string>50</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QRadioButton" name="rbAxis_1" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>60</y>
      <width>111</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text" >
     <string>X Axis</string>
    </property>
    <property name="checked" >
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_3" >
    <property name="geometry" >
     <rect>
      <x>190</x>
      <y>90</y>
      <width>50</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text" >
     <string>um</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rbAxis_2" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>90</y>
      <width>111</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text" >
     <string>Y Axis</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtPadVal_2" >
    <property name="geometry" >
     <rect>
      <x>122</x>
      <y>90</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text" >
     <string>5000</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtPadVal_1" >
    <property name="geometry" >
     <rect>
      <x>122</x>
      <y>60</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text" >
     <string>5000</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2" >
    <property name="geometry" >
     <rect>
      <x>190</x>
      <y>60</y>
      <width>50</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text" >
     <string>um</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="chkBar_3" >
    <property name="geometry" >
     <rect>
      <x>180</x>
      <y>20</y>
      <width>101</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text" >
     <string>BarChar</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_5" >
   <property name="geometry" >
    <rect>
     <x>330</x>
     <y>270</y>
     <width>311</width>
     <height>121</height>
    </rect>
   </property>
   <property name="title" >
    <string>Angle Parameter</string>
   </property>
   <widget class="QWidget" name="layoutWidget_5" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>20</y>
      <width>159</width>
      <height>22</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QLabel" name="label_11" >
       <property name="text" >
        <string>Scale-Z</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="txtScaleZ_4" >
       <property name="text" >
        <string>30</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QCheckBox" name="chkBar_4" >
    <property name="geometry" >
     <rect>
      <x>180</x>
      <y>20</y>
      <width>101</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text" >
     <string>BarChar</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_6" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>140</y>
     <width>311</width>
     <height>121</height>
    </rect>
   </property>
   <property name="title" >
    <string>Focus Parameter</string>
   </property>
   <widget class="QWidget" name="layoutWidget_6" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>20</y>
      <width>281</width>
      <height>22</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QLabel" name="label_12" >
       <property name="text" >
        <string>Scale-Z</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="txtScaleZ_1" >
       <property name="text" >
        <string>800</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="chkBar_1" >
       <property name="text" >
        <string>BarChar</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton_2" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>400</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text" >
    <string>Load(&amp;L)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3" >
   <property name="geometry" >
    <rect>
     <x>100</x>
     <y>400</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text" >
    <string>Save(&amp;S)</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <resources/>
 <connections/>
</ui>
